### PR TITLE
lesson_1_6

### PR DESCRIPTION
**Pull request by AigulY**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,12 +20,17 @@ function App() {
         setTodoList([...todoList, newTodo]);
     };
 
+    const removeTodo = (id) => {
+        const updatedTodoList = todoList.filter(todo => todo.id !== id);
```
Comment (JoHeyJo) [inline src/App.js:24]: Nice one-liner!

File: src/InputWithLabel.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from 'react';
+
+const InputWithLabel = (props) => {
+    const inputRef = useRef(null);
```
Comment (JoHeyJo) [inline src/InputWithLabel.js:4]: Nice use of `null`.

File: src/InputWithLabel.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from 'react';
+
+const InputWithLabel = (props) => {
```
Comment (JoHeyJo) [inline src/InputWithLabel.js]: Let's be consistent with our coding pattern. Throughout your app you've been destructuring your props however here you are passing the prop object. Either choice is fine. Being consistent is good practice.

**Review by JoHeyJo — APPROVED** [review]
Nicely done! Just one comment that needs to be addressed and then go ahead and merge it!

**Review by AigulY — COMMENTED** [review]
destructuring the props

**Pull request merged**